src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -38,17 +37,13 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     @PutMapping("/movie-sessions")
     public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
         shoppingCartService.addSession(
-                movieSessionService.get(movieSessionId), getUser(authentication));
+                movieSessionService.get(movieSessionId),
+                userService.findByEmail(authentication.getName()).get());
     }
 
     @GetMapping("/by-user")
     public ShoppingCartResponseDto getByUser(Authentication authentication) {
-        User user = getUser(authentication);
+        User user = userService.findByEmail(authentication.getName()).get();
```

Comment:
Throwing a new EntityNotFoundException was good idea)
it is ok to have two methods in UserService:
```
Optional<User> findByEmail(String email);
User getByEmail(String email);
```